docs/developer-guide/02-projects/image-builder/index.md

```diff
@@ -0,0 +1,50 @@
+Image Builder
+=============
+
+Image Builder serves as an HTTP API on top of [Osbuild
+Composer](https://github.com/osbuild/osbuild-composer), and serves as the
+backend for [Image Builder
+Frontend](https://github.com/osbuild/image-builder-frontend/).
```

Comment:
we could consult readme-list and change those easy known links to local ones ;)
should I check if that's easy?

Reply:
hmm... maybe the "Projects" page could contain the links to the sources/original readmes... 🤔